graphql/schema/types/host.graphql

```diff
@@ -22,6 +22,9 @@ enum TaskQueueItemType {
 type SleepSchedule {
   dailyStartTime: String!
   dailyStopTime: String!
+  isBetaTester: Boolean!
```

Comment:
Just sanity checking — will you need `isBetaTester` in the input type as well? Will it be set by a toggle on the UI?

Reply:
You just saved me a PR 🙏 thanks!